src/main/webapp/src/components/FeedbackForm.vue

```diff
@@ -0,0 +1,220 @@
+                        class="feedback-label"
+                        >Wie lautet Dein Name?</label
+                    >
+                    <input
+                        type="text"
+                        v-model="$v.feedback.participantName.$model"
+                        @change="$emit('feedback', feedback)"
+                        id="feedback-participant-name"
+                        placeholder="Kurzantwort-Text (Optional)"
+                    />
+                    <span
+                        v-if="$v.feedback.participantName.$error"
+                        class="feedback-validation-text"
+                        >Die maximale Länge sind 255 Zeichen.
+                    </span>
+                </div>
+            </div>
+            <div class="row">
+                <div class="column">
+                    <label for="feedback-dislikes" class="feedback-label"
+                        >Was hat Dir im Rahmen der Fortbildung nicht
+                        gefallen?</label
+                    >
+                    <input
+                        type="text"
+                        v-model="$v.feedback.dislikes.$model"
+                        @change="$emit('feedback', feedback)"
+                        id="feedback-dislikes"
+                        placeholder="Langantwort-Text (Optional)"
+                    />
+                    <span
+                        v-if="$v.feedback.dislikes.$error"
+                        class="feedback-validation-text"
+                        >Die maximale Länge sind 2000 Zeichen.
+                    </span>
+                </div>
+            </div>
+            <div class="row">
+                <div class="column">
+                    <label for="feedback-likes" class="feedback-label"
+                        >Was hat Dir besonders gut gefallen?</label
+                    >
+                    <input
+                        type="text"
+                        v-model="$v.feedback.likes.$model"
+                        @change="$emit('feedback', feedback)"
+                        id="feedback-likes"
+                        placeholder="Langantwort-Text (Optional)"
+                    />
+                    <span
+                        v-if="$v.feedback.likes.$error"
+                        class="feedback-validation-text"
+                        >Die maximale Länge sind 2000 Zeichen.
+                    </span>
+                </div>
+            </div>
+            <div class="row">
+                <div class="column">
+                    <label
+                        :class="
+                            validationStateClass('feedback.recommendation') +
+                                ' feedback-label required-label'
+                        "
+                        >Kannst Du die Teilnahme an dieser Fortbildung
+                        weiterempfehlen?</label
+                    >
+                    <div class="feedback-radio-button-container">
```

Comment:
The selection circle in the selected radio button is not centered

Reply:
good point. that is on firefox. I did not notice it since i am using chrome :)